tips/025_tip_create_new_authority.md

```diff
@@ -34,29 +38,36 @@ Do not forget to include your new role in your authorities.csv:
 	ROLE_ADMIN
 	ROLE_USER
 	ROLE_ANONYMOUS
+	ROLE_LOCAL
 
 
-With that, you will be able to use it in your SecurityConfiguration.java or in (FrontpageconfigResource.java), for example:
+With that, you will be able to use it in your SecurityConfiguration.java or in (UserResource.java), for example:
 	
-	@DeleteMapping("/order-items/{id}")
-	@Timed
-	@PreAuthorize("hasAuthority('ROLE_ADMIN')")
-	public ResponseEntity<Void> deleteOrderItem(@PathVariable Long id) {
-	    ...
-	}
+    /**
+     * Gets a list of all roles.
+     * @return a string list of all roles.
+     */
+    @GetMapping("/users/authorities")
+    @PreAuthorize("hasAuthority(\"" + AuthoritiesConstants.ADMIN + "\")")
+    public List<String> getAuthorities() {
+        return userService.getAuthorities();
+    }
 
-and/or Angular files: jhiHasAnyAuthority=[‘ROLE_ADMIN’. ‘ROLE_X’ ……] or even consider to use it in the routes:
-
-	export const messageRoute: Routes = [
-	    {
-	        path: 'message',
-	        component: MessageComponent,
-	        data: {
-	            authorities: ['ROLE_USER'],
-	            pageTitle: 'Messages'
-	        },
-	        canActivate: [UserRouteAccessService]
-	    }
-	];
+and/or in your Angular files: jhiHasAnyAuthority=[‘ROLE_ADMIN’. ‘ROLE_X’ ……] or even consider to use it in the routes:
+
+export const appuserRoute: Routes = [
```

Comment:
This code section needs to be indented to put the code in a pre-formatted block

You can preview the changes by clicking on the page icon at the top ([screenshot](https://user-images.githubusercontent.com/4294623/80382880-d9cc9700-8870-11ea-8ffc-e193dc29347e.png)) or for the current changes: https://github.com/jhipster/jhipster.github.io/pull/949/files?short_path=a2064be#diff-a2064bef6385b464cd1f30572c8c881f

Reply:
Thanks Jon